Cannot convert FauxZero

Cloned repository and using the quick start instructions.

 $ go run ./pkg/v1/agent/deepq/experiments/cartpole/main.go
 \# github.com/aunum/gold/pkg/v1/dense
 pkg/v1/dense/value.go:57:19: cannot convert FauxZero (type float64) to type complex64
 pkg/v1/dense/value.go:59:20: cannot convert FauxZero (type float64) to type complex128

Using go v1.14 on linux/amd64

--- a/pkg/v1/dense/value.go
+++ b/pkg/v1/dense/value.go
@@ -53,10 +53,11 @@ func FauxZeroValue(dt t.Dtype) interface{} {
 		return float32(FauxZero)
 	case reflect.Float64:
 		return float64(FauxZero)
-	case reflect.Complex64:
-		return complex64(FauxZero)
-	case reflect.Complex128:
-		return complex128(FauxZero)
+	// TODO: this breaks in 1.14
+	// case reflect.Complex64:
+	// 	return complex64(FauxZero)
+	// case reflect.Complex128:
+	// 	return complex128(FauxZero)
 	default:
 		panic(fmt.Sprintf("type not supported: %#v", dt))
 	}

